src/components/views/rooms/ReplyPreview.tsx

```diff
@@ -59,3 +58,7 @@ export default class ReplyPreview extends React.Component<IProps> {
         );
     }
 }
+
+export default forwardRef<ReplyPreview, Omit<IProps, "context">>((props, ref) => (
+    <RoomContext.Consumer>{(context) => <ReplyPreview {...props} context={context} ref={ref} />}</RoomContext.Consumer>
+));
```

Comment:
How come MatrixClientContext qualifies for a HOC to do this boilerplate, but the 15000 `<RoomContext.Consumer>`s don't, by the way?